aws-lambda-java-runtime-interface-client/test/integration/codebuild-local/codebuild_build.sh

```diff
@@ -95,7 +97,7 @@ then
     exit 1
 fi
 
-docker_command="docker run "
+docker_command="docker run -it "
```

Comment:
I think interactive mode only works with TTY, I believe this is the cause of the checks failing - any reason we need `-it` here?

Reply:
Actually no we don't, it was just copied from upstream as it is. I will remove it.